### PR TITLE
restore support for Windows cuda 11

### PR DESCRIPTION
**Pull request by tlambert03**
in #29, I dropped support for cuda 11 on Windows, so we could at least bring *something* up to date.  It works now for linux on cuda11/12 and windows cuda12.  This PR is trying to bring back support for windows cuda 11.  Going to start with a broken build.

### CODE REVIEW

**Comment by jakirkham** [comment]
Maybe we need [this logic]( https://github.com/conda-forge/cudasirecon-feedstock/blob/1aa45ec845a626ba331cbfa3c98635426745de3c/recipe/bld.bat#L4-L7 ):

```batch
:: Handle external CTK in CUDA 11 builds
if defined CUDA_PATH (
    SET CUDACXX=%CUDA_PATH%\bin\nvcc.exe
)
```

**Comment by tlambert03** [comment]
well i was very hopeful!  (and going to be somewhat embarrassed) ... but it looks like no: https://dev.azure.com/talleylambert/talleylambert/_build/results?buildId=81&view=logs&j=5be07ae1-d8ba-5406-47b6-8e3a3a12f825&t=c94d5231-acfc-5e67-dafc-77bdd383ead4&l=1129

**Comment by jakirkham** [comment]
Admittedly that logic was from `cudasirecon`. Don't think `cudadecon` has used something similar before

Do we see the same issue with CUDA 11.8? In conda-forge, we [bumped the minimum to CUDA 11.8]( https://conda-forge.org/news/2024/03/06/dropping-cuda-112/ ). So maybe this is just a hang up due to a deprecated CUDA version

**Comment by jakirkham** [comment]
Think we need a re-render too as it appears to be installing CUDA 11.2

**Comment by jakirkham** [comment]
Looks like this needs an update to CUDA 11.8 as well

https://github.com/scopetools/cudadecon/blob/02dd3a8b6c2d5a801c38245fd0bf6816e9385659/recipe/conda_build_config.yaml#L1-L2

Would also re-render after making that change

Should fix the Linux CI error

**Comment by tlambert03** [comment]
Yep!  Success!  Will do tomorrow

**Comment by jakirkham** [comment]
Indeed. Glad to see this was relatively simple to fix 😄

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -14,19 +14,19 @@ cdt_name:                 # [linux]
...
   - cuda-nvcc
 
 cuda_compiler_version:
-  - 11.2                  # [not win]
+  - 11.8
   - 12.0
 
 libboost_devel:           
-  - '1.82'                # [not win]             
+  - '1.82'
```
Comment (jakirkham) [inline recipe/conda_build_config.yaml:25]: Looks like there was some leftover blank space

```suggestion
  - '1.82'
```

**Comment by jakirkham** [comment]
Thanks Talley! 🙏

Would it make sense to cut a new release?

Or do you want to finish PR ( https://github.com/scopetools/cudadecon/pull/34 ) first?

**Comment by tlambert03** [comment]
I'm working on a few other things with this repo this week and will cut a new release shortly.  thanks again for all your help here

**Comment by jakirkham** [comment]
No worries. Thanks for all of your efforts here! 🙏

**Pull request merged**